misk-hibernate/src/test/kotlin/misk/hibernate/ReflectionQueryFactoryTest.kt

```diff
@@ -253,63 +258,63 @@ class ReflectionQueryFactoryTest {
    */
   @Test
   fun rowResultCountWarning() {
-    // 1900 rows is too small for the warning threshold (2000).
+    // 18 rows is too small for the warning threshold (20).
     transacter.transaction { session ->
-      for (i in 1..1900) {
+      for (i in 1..18) {
         session.save(DbMovie("Rocky $i", null))
       }
     }
     transacter.transaction { session ->
-      assertThat(queryFactory.newQuery<OperatorsMovieQuery>().list(session)).hasSize(1900)
+      assertThat(queryFactory.newQuery<OperatorsMovieQuery>().list(session)).hasSize(18)
     }
     assertThat(logCollector.takeMessages(loggerClass = ReflectionQuery::class)).isEmpty()
 
-    // 2100 rows logs a warning.
+    // 21 rows logs a warning.
     transacter.transaction { session ->
-      for (i in 1901..2100) {
+      for (i in 19..21) {
         session.save(DbMovie("Rocky $i", null))
       }
     }
     transacter.transaction { session ->
-      assertThat(queryFactory.newQuery<OperatorsMovieQuery>().list(session)).hasSize(2100)
+      assertThat(queryFactory.newQuery<OperatorsMovieQuery>().list(session)).hasSize(21)
     }
     assertThat(getOnlyElement(
         logCollector.takeMessages(loggerClass = ReflectionQuery::class, minLevel = Level.WARN))
-    ).startsWith("Unbounded query returned 2100 rows.")
+    ).startsWith("Unbounded query returned 21 rows.")
 
-    // 3100 rows logs an error.
+    // 31 rows logs an error.
     transacter.transaction { session ->
-      for (i in 2101..3100) {
+      for (i in 22..31) {
         session.save(DbMovie("Rocky $i", null))
       }
     }
     transacter.transaction { session ->
-      assertThat(queryFactory.newQuery<OperatorsMovieQuery>().list(session)).hasSize(3100)
+      assertThat(queryFactory.newQuery<OperatorsMovieQuery>().list(session)).hasSize(31)
     }
     assertThat(getOnlyElement(
         logCollector.takeMessages(loggerClass = ReflectionQuery::class, minLevel = Level.ERROR))
-    ).startsWith("Unbounded query returned 3100 rows.")
+    ).startsWith("Unbounded query returned 31 rows.")
 
     // An explicit max row count suppresses the warning.
     transacter.transaction { session ->
       assertThat(queryFactory.newQuery<OperatorsMovieQuery>()
-          .apply { maxRows = 3200 }
+          .apply { maxRows = 32 }
           .list(session))
-          .hasSize(3100)
+          .hasSize(31)
     }
     assertThat(logCollector.takeMessages(loggerClass = ReflectionQuery::class)).isEmpty()
 
-    // More than 10000 rows throws an exception.
+    // More than 40 rows throws an exception.
     transacter.transaction { session ->
-      for (i in 3101..10100) {
+      for (i in 32..41) {
```

Comment:
Were any of these tests (that have been significantly reduced in quantity) the stress testing ones that had been mentioned in discussions? If so, should we have a way to still run them periodically (say daily) but not on every build / PR?

Reply:
The intention of these aren't to stress test, they're to see that the various logs are outputted

Reply:
(I wouldn't consider 10K to necessarily be "stress testing" anyways)